molecule/converge.yml

```diff
@@ -3,8 +3,12 @@
   become: true
   vars:
     nox_version: "0.22.0"
-    ipfs_cli_version: "0.25.0"
     ccp_version: "0.2.0"
+    ipfs_cli_version: "0.25.0"
```

Comment:
0.26.0?

Reply:
Nah doesn't matter. It is just for tests.